src/test/java/org/opentripplanner/updater/vehicle_sharing/vehicle_presence/CarPresencePredictorTest.java

```diff
@@ -0,0 +1,95 @@
+package org.opentripplanner.updater.vehicle_sharing.vehicle_presence;
+
+import junit.framework.TestCase;
+import org.locationtech.jts.geom.Envelope;
+import org.opentripplanner.prediction_client.VehiclePresence;
+import org.opentripplanner.routing.core.vehicle_sharing.CarDescription;
+import org.opentripplanner.routing.core.vehicle_sharing.FuelType;
+import org.opentripplanner.routing.core.vehicle_sharing.Gearbox;
+
+import java.util.Collections;
+import java.util.List;
+
+public class CarPresencePredictorTest extends TestCase {
```

Comment:
junit4 please

Reply:
I don't remember what happened here :) I recall trying to make parametrised test